conda-recipes/ray-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "ray" %}
+{% set version = "0.7.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ray-project/ray.git
+  git_rev: ray-{{ version }}
+  patches:
+    - redis-bazel-build.patch
+    
+build:
+  noarch: python
+  number: 0
+  rpaths_patcher: patchelf
+
+requirements:
+  host:
+    - python
+    - cython==0.29.0
+    - openssl
+    - bazel
+    - arrow
+    - pip
+    - pyarrow
+    - git
+    - cmake
+    - make
+    - setuptools
+    - protobuf==3.8.0
```

Comment:
More @jayfurmanek ... This protobuf dependency limits (maybe) this package to being usable only with WML CE 1.6.2 (1.6.1 used protobuf 3.7.1). Maybe we don't care about the looking back to 1.6.1, but probably should think about how to handle this for the future.

Note that if the build pulls in _our_ protobuf package (and it should, based on channel priority), I don't think we include a `run_exports` that would automatically limit the `run` dependency to same protobuf version. (I just coded `host / protobuf=3.7.1` to build a 1.6.1-compatible package and it installed just fine (though presumably would be broken at runtime) with protobuf 3.8.0.)  So we maybe should code a `pin_compatible` or something for the `run:` / `protobuf` req.

Reply:
I tried to use other versions of protobuf and got some errors (at runtime if I remember well), that's why I pinned v 3.8.0, and was using conda-forge channel
Now that WMLCE 1.6.2 has protobuf=3.8.0, I removed the conda-forge channel from the build